src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepository.scala

```diff
@@ -25,13 +23,22 @@ class ConsignmentRepository(db: Database) {
     db.run(query.result)
   }
 
+  def getSeriesAndBodyOfConsignment(consignmentId: UUID)(implicit executionContext: ExecutionContext): Future[Seq[ConsignmentResult]] = {
```

Comment:
Now that we're using separate resolvers for the transferring body and the series, I think it's worth splitting this up into two separate repository methods.

It means more code, but I think it's less confusing for future developers because they won't have to work out why the series resolver gets the transferring body details and then throws them away.

Reply:
Ok, sure!